src/tests/Microsoft.PowerFx.Interpreter.Tests/ThreadingTests.cs

```diff
@@ -20,5 +23,48 @@ public void CheckInterpreter()
 
             AnalyzeThreadSafety.CheckStatics(asm, bugsFieldType, bugNames);
         }
+
+        // $$$ Supersedes ImmutabilityTests. This is more aggressive (incldues private fields). 
+        [Fact]
+        public void CheckImmutableType()
```

Comment:
Run this test under a debugger, and failure list is written to Debugger output window.

Reply:
can you add this comment in the file as well so anyone run know where to look for?